### ISSUE TITLE
Immortal Entity Clone

### ISSUE THREAD

**Issue by Mimero** [iChun/Morph#1200]
Minecraft Version: 1.16.5
Modpack: Project Architect
Mode: Survival Singleplayer
Issue: Entering a Netherportal from the Overworld morphed as Ghast spawns a immortal Entity Clone of the Player.

**Comment by iChun** [comment]
And this doesn't happen when not morphed? Morph doesn't actually
remove/spawn entities when you're morphed, you're still a player, you just
don't look like one.

**Comment by Mimero** [comment]
It fine with every other morph so far. Just the Ghast seems to bug it out.
"Restarting the game doesnt despawn it and it wasnt killable via commands.

**Comment by LunarDev-git** [comment]
Same is happening with me and for me it's an Enderman

1.16.5

**Comment by LunarDev-git** [comment]
@Mimero If you're still having the issue, I managed to remove the bugged entity with Crash Utilities mod.

**Comment by iChun** [comment]
Do you know what kind of entity is being spawned? I can't seem to reproduce this in my own tests.

Was this in Multiplayer/Singleplayer? Like I mentioned before, Morph doesn't spawn any entities, to the server and client, you're still a player, you just look/sound like a different entity.

I'm curious if this is a mod conflict in the mod pack

**Comment by iChun** [comment]
Does this also happen exclusively on the Project Architect mod pack?

**Comment by LunarDev-git** [comment]
So, I was using a self created kitchen sink modpack unlike the original poster. 
That being said, I come baring great news which is I managed to recreate the bug after spending an hour trying to reproduce it, I had only 3 mods enabled, listed below.

**Issue:** Unkillable clone of myself/the entity I was morphed as. That you cannot remove with /kill. The only way I managed to remove it is by using Crash Utilities and force removing listed entities. The clone is not just a client side visual bug. 

**Mods enabled:** iChunUtils & Morph. (Optifine and crash utilities but it happens without it)
**Gamemode the bug happens on:** Survival Singleplayer & Survival Multiplayer. 
**Version**: 1.16.5
**Mod Version**: Morph-1.16.5-10.1.1 & iChunUtil-1.16.5-10.5.2

**How to reproduce:** Morph into an Enderman, make a 3x3x2 pool of water, go in there and keep disconnecting and reconnecting until you see an enderman appear. _(Unsure if the size of the pool of water has any effect but that's what I used during my testing, so trying to make as little variables as possible for when you happen to test)_

[Video evidence. ](https://www.youtube.com/watch?v=mgqlF16T1GI)

_Additional Notes:_ I don't know how OP created a Ghast clone but at least I found this bug which may help aid the situation down the line with other entity cloning.

**Comment by iChun** [comment]
I think I figured it out, it's due to a very edge-case where an entity changes chunks on it's own accord (despite having no AI) and when the world reloads, it causes the entity to spawn. 

I should have been able to address this and these mobs should no longer spawn.

I've also made it so that in the event that it does happen again, the /kill command should damage them (and kill them) appropriately. Look out for this fix in the next update!

Thanks for the steps to reproduce, those helped immensely.
